spec/features/admin/sign_out_spec.rb

```diff
@@ -15,9 +15,28 @@
   end
 
   scenario 'allows a signed in user to logout' do
-    click_link 'Logout'
+    perform_logout
     visit spree.admin_login_path
     expect(page).to have_text 'Login'
     expect(page).not_to have_text 'Logout'
   end
+
+  def perform_logout
```

Comment:
nit: thoughts on renaming it `click_logout_link`? That would be more in line with existing capybara helpers (e.g. `click_link`)